benchmark/directory.dart

```diff
@@ -4,24 +4,28 @@
 
 import 'dart:io';
 
+import 'package:args/args.dart';
 import 'package:dart_style/dart_style.dart';
 import 'package:dart_style/src/constants.dart';
 import 'package:dart_style/src/profile.dart';
+import 'package:dart_style/src/testing/benchmark.dart';
 
 /// Reads a given directory of Dart files and repeatedly formats their contents.
 ///
 /// This allows getting profile information on a large real-world codebase
 /// while also factoring out the file IO time from the process.
-void main(List<String> args) {
-  if (args.length != 1) {
-    stderr.writeln('Usage: dart run directory.dart <directory to format>');
-    exit(65);
-  }
+void main(List<String> arguments) async {
+  var directory = await _parseArguments(arguments);
 
-  var directory = args[0];
   print('Listing entries in $directory...');
   var entries = Directory(directory).listSync(recursive: true);
 
+  // Make sure the benchmark is deterministic. The order shouldn't really
+  // matter for performance, but since the JIT is warming up as it goes through
+  // the files, different orders could potentially affect how it chooses to
+  // optimize.
+  entries.sort((a, b) => a.path.compareTo(b.path));
```

Comment:
[optional] If you add an import to `package:collection`

```suggestion
  entries.sortBy((e) => e.path);
```

Reply:
TIL. Done!